js/component-courses.js

```diff
@@ -1,2 +1,40 @@
+
+// let courses = '<ul id="app-course">' +
+//         '<li v-for="course in list">{{ course }}</li>' +
+//         '</ul>';
+// let list;
+
+// /* Fetch the returned json object value. */
+// fetch('get_info.php').then(function (response) {
+//     list = JSON.parse(response);
+// });
+
 // export as global since we're not using a module system
-window.Courses = { template: '<div>Classes</div>' }
+window.Courses = Vue.component('Courses', {
+    name: "Courses",
+    data: function () {
+        return {list:null};
```

Comment:
Run your code through the prettier.js code formatter with indentation at 4 spaces.

Reply:
Thanks, I started using prettier!